examples/gauss_markov_process_example.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2021 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# You can plot the data generated by this program by following these
+# steps.
+#
+# 1. Run this program and save the output to a file:
+#     ./gauss_markov_process > plot.data
```

Comment:
I think we should tell the user to set `PYTHONPATH` like we do on other examples:

https://github.com/ignitionrobotics/ign-math/blob/1cac6547f141e5f5872d8d2976ebc1ac50633135/examples/angle_example.py#L18-L21

```suggestion
#     python3 gauss_markov_process_example.py > plot.data
```